.github/workflows/docs-ci.yaml

```diff
@@ -15,9 +15,13 @@ on:
 
 jobs:
   docs-ci:
+    permissions:
+      # need to provide GITHUB_TOKEN with content/pull-request write premissions
+      contents: write
+      pull-requests: write  
     runs-on: ubuntu-latest
     env:
-      GH_TOKEN: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
are we using this env var in the job ?

Reply:
@adrianchiris see my comment from the original line 33 in this file:
> # token must be explicitly set here for push to work in following step

@heyvister1 , although you renamed the secret variable ([in line 33](https://github.com/Mellanox/network-operator-docs/pull/134/files#diff-b1d6bf418157ab5fbc536f483dd7fae79a274c50824d70aae62de229c2023140R37)) - perhaps it's worth preserving the comment (unless you tested that after your change the token no longer needs to be explicitly set?).